src/mlpack/methods/pca/pca_impl.hpp

```diff
@@ -1,19 +1,29 @@
 /**
  * @file pca.cpp
  * @author Ajinkya Kale
+ * @author Ryan Curtin
+ * @author Marcus Edel
  *
  * Implementation of PCA class to perform Principal Components Analysis on the
  * specified data set.
  */
-#include "pca.hpp"
+
+#ifndef MLPACK_METHODS_PCA_PCA_IMPL_HPP
+#define MLPACK_METHODS_PCA_PCA_IMPL_HPP
+
 #include <mlpack/core.hpp>
+#include "pca.hpp"
 
 using namespace std;
```

Comment:
It's used for:

``
Log::Fatal << "PCA::Apply(): varRetained (" << varRetained << ") must be "
        << "greater than or equal to 0." << endl;
``